geronimo-openapi-impl/src/main/java/org/apache/geronimo/microprofile/openapi/impl/processor/SchemaProcessor.java

```diff
@@ -145,45 +146,22 @@ public void fillSchema(
                     fillSchema(components, Object.class, items, null);
                     schema.items(items);
                 } else {
-                    ofNullable(from.getAnnotation(Schema.class)).ifPresent(
-                            s -> sets(components, Schema.class.cast(s), schema, null));
+                    Optional<Annotation> annotation = ofNullable(from.getAnnotation(Schema.class));
```

Comment:
Can be typed Optional[Schema] to avoid the casts some lines later or just keep the getAnnotation result on previous line?

Reply:
Use now Optional[Schema]